src/src/tx_main.cpp

```diff
@@ -583,9 +586,11 @@ void ICACHE_RAM_ATTR SendRCdataToRF()
     // set the transceiver the correct fallback mode
     Radio.TXdoneCallback();
   }
+  else
```

Comment:
TXdoneCallback starts CCA for the next packet.  But if Radio.TXnb is immediately called afterwards the radios are placed into STBY, which stops the instantaneous rssi updates.